securesystemslib/keys.py

```diff
@@ -474,7 +474,7 @@ def format_keyval_to_metadata(keytype, scheme, key_value, private=False):
 
 
 
-def format_metadata_to_key(key_metadata, default_keyid=None):
+def format_metadata_to_key(key_metadata, default_keyid=None, hash_algorithms=securesystemslib.settings.HASH_ALGORITHMS):
```

Comment:
The [coding style](https://github.com/secure-systems-lab/code-style-guidelines) requests we keep lines to 79 characters or less with a double (4 space) indentation for continuation. Could you wrap this line?

```suggestion
def format_metadata_to_key(key_metadata, default_keyid=None,
    hash_algorithms=securesystemslib.settings.HASH_ALGORITHMS):
```

Reply:
Yes, you are right. It seems I overlooked it.

Fixed and rebased on the master branch.